### PR TITLE
Jlechowicz/issue 297

### LINKED ISSUE #297: SSN field usability updates
- Dashes should be input automatically
- Users can only enter numbers
- Design should look like 1a and 1b attached in a comment below

### PR DESCRIPTION
**Pull request by jlechowicz**
Added masking for SSN field and added a show/hide toggle.
===

Resolves #297 

- [X] Snapshots updated, if output HTML/JS has changed (`npm run test:update-snapshots`)
- [ ] Changes reviewed on mobile using devtools, if output HTML/CSS has changed
- [ ] Spanish checked by adding `?lng=es` to URL e.g. `/guide/benefits?lng=es`, if Spanish updated
- [ ] Automated tests added, if new functionality added
- [ ] Documentation (e.g. READMEs) updated, if relevant
- [ ] IE11 and Edge compatibility confirmed via manual testing in Browserstack, if new libraries added or newish JS/HTML/CSS features used for the first time in this codebase
- [ ] Accessibility & performance audit run using Google Lighthouse, if major changes made

### CODE REVIEW

File: setupTestFramework.js [hunk 1]
```diff
@@ -45,4 +45,4 @@ jest.mock("uuid", () => {
     ...actual,
     v4: () => "00000000-fake-mock-fake-123456789012",
   };
-})
\ No newline at end of file
+});
```
Comment (idealisms) [inline setupTestFramework.js:48]: Thanks for fixing!

File: public/locales/es/translation.json [hunk 2]
```diff
@@ -197,6 +197,10 @@
     "last-name-required-error": "Please provide your Last name.",
     "eddcan-required-error": "Please provide your EDDCAN number.",
     "ssn-required-error": "Please provide your SSN.",
-    "missing-fields-error":"It looks like you missed some information. Please fill out all fields and try again."
-  }
+    "missing-fields-error":"It looks like you missed some information. Please fill out all fields and try again.",
+    "show-ssn":"Show SSN",
+    "hide-ssn":"Hide SSN"
+  },
+  "show":"Show",
```
Comment (sharonwarner) [inline public/locales/es/translation.json:204]: I think we can remove these fields.

Comment (jlechowicz) [reply public/locales/es/translation.json]: Removed

**Comment by jlechowicz** [comment]
@idealisms It doesn't look like we can move the cursor to the middle (Unless there is something already there). If it turns about to be am issue for UX we can circle back on this.

**Pull request merged**